Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'will_paginate-bootstrap'
 gem 'yt'
 gem 'yui-compressor'
 gem 'youtube_rails'
+gem 'google-api-client', '0.8'
```

Comment:
@mattlindsey That would require lot of time because I was able to pull up this one from existing examples given here https://github.com/youtube/api-samples/tree/master/ruby but there are no such examples for newest version.